custom.example.css

```
@@ -21,8 +28,8 @@
 .frame--layout-splitview:nth-child(1){}
 .frame--layout-splitview:nth-child(2){}
 .frame--layout-splitview .frame--image{}
+.layout-splitview-landscape{}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)